spec/features/user_can_reply_to_a_twit_spec.rb

```diff
@@ -0,0 +1,71 @@
+require "rails_helper"
+
+RSpec.feature "Reply to a Twit", type: :feature do
+  before do
+    # Short circuit OmniAuth requests to use a mock authentication hash
+    OmniAuth.config.test_mode = true
+
+    OmniAuth.config.mock_auth[:twitter] = OmniAuth::AuthHash.new({
+      :provider => "twitter",
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Use 2 spaces for indentation in a hash, relative to the first position after the preceding left parenthesis.